dia/diameter_nasreq_rfc7155.dia

```diff
@@ -0,0 +1,341 @@
+@id	1
```

Comment:
Please adjust to formatting the to match that of the DIAMETER dictionaries shipped with OTP, see https://github.com/erlang/otp/blob/master/lib/diameter/src/dict/base_rfc6733.dia.

Add a copyright header that is consistent with the other files in this project,
 e.g. https://github.com/travelping/ergw_aaa/blob/master/src/ergw_aaa.erl#L1